app_remote/tests/test_sidebar.py

```diff
@@ -0,0 +1,4 @@
+def test_sidebar(client):
+    actual = client.get("/sidebar")
+    assert actual.status_code == 200
+    assert actual.data is not None
```

Comment:
why not test the actual content?

Reply:
Content will change soon as we are going to add a `redaction` button and possibly more. Testing actual data may require us to constantly update the test reflecting the change we made. Just wondered is it a convention for people to test the actual content in web development?

Reply:
I think it's good practise to check the response body. maybe just look for sub-strings and don't check the whole html text, in order to minimise updates when changes occur. But it's good to at least smoke test to make sure that this endpoint doesn't return garbage.